docs/resources/Audit_Log.md

```diff
@@ -126,12 +126,12 @@ Whenever an admin action is performed on the API, an entry is added to the respe
 | nick | [user](#DOCS_USER/user-object) | string | user nickname changed |
 | avatar_hash | [user](#DOCS_USER/user-object) | string | user avatar changed |
 | id | any | snowflake | the id of the changed entity - sometimes used in conjunction with other keys |
-| type | any | string/int | type of entity created - can be of [channel type](#DOCS_CHANNEL/channel-object-channel-types) or a string like "role" |
+| type | any | integer ([channel type](#DOCS_CHANNEL/channel-object-channel-types)) or string | type of entity created |
```

Comment:
From what I could find, yes. I will double check to be sure, and am open to being shown otherwise.